src/main/java/programminglife/gui/controller/GraphController.java

```diff
@@ -56,13 +53,25 @@ public void draw(int centerNode, int radius) {
                 drawEdge(drawableNode, child);
             }
         }
+
+        DrawableNode drawableCenterNode = subGraph.getNodes().get(centerSegment);
+        drawableCenterNode.setFill(Color.ORANGE);
+        double xCoord = drawableCenterNode.getX();
+        Bounds bounds = grpDrawArea.getParent().getLayoutBounds();
+        double boundsHeight = bounds.getHeight();
+        double boundsWidth = bounds.getWidth();
+
+        grpDrawArea.setTranslateX(boundsWidth / 2 - xCoord);
+        grpDrawArea.setTranslateY(boundsHeight / 3);
```

Comment:
This should be in a separate method, because it can also be used in reset x/y for example.